libraries/AP_RangeFinder/AP_RangeFinder_Params.h

```diff
@@ -24,7 +24,11 @@ class AP_RangeFinder_Params {
     AP_Int16 min_distance_cm;
     AP_Int16 max_distance_cm;
     AP_Int8  ground_clearance_cm;
+    AP_Int8  glitch_count;
+    AP_Int16 glitch_distance_cm;
     AP_Int8  address;
     AP_Vector3f pos_offset; // position offset in body frame
     AP_Int8  orientation;
+    AP_Float flt_coeff;
+    AP_Int8  glitchcount;
```

Comment:
looks like we have two extra parameters here vs the parameter table above.  i.e. "glitchcount" and "glitch_count".  Also I don't see a param in the parameter table for "glitch_distance_cm".